demisto_sdk/commands/init/tests/initiator_test.py

```diff
@@ -181,3 +184,75 @@ def test_yml_reformatting(tmp_path, initiator):
             'display': 'HelloWorld',
             'name': 'HelloWorld'
         })
+
+
+@patch('demisto_sdk.commands.split_yml.extractor.get_python_version')
+@patch('demisto_sdk.commands.init.initiator.get_content_path')
+def test_convert_contribution_zip(get_content_path_mock, get_python_version_mock, tmp_path, initiator):
+    '''Create a fake contribution zip file and test that it is converted to a Pack correctly
+
+    Args:
+        get_content_path_mock (MagicMock): Patch of the 'get_content_path' function to return the fake repo directory
+            used in the test
+        get_python_version_mock (MagicMock): Patch of the 'get_python_version' function to return the "3.7"
+        tmp_path (fixture): Temporary Path used for the unit test and cleaned up afterwards
+        initiator (fixture): Initializes an instance of the 'Initiator' class
+
+    Scenario: Simulate executing the 'init' command with the 'contribution' option passed
+
+    Given
+    - A contribution zip file
+    - The zipfile contains a unified script file
+    - The zipfile contains a unified integration file
+    When
+    - Converting the zipfile to a valid Pack structure
+    Then
+    - Ensure script and integration are componentized and in valid directory structure
+    '''
+    # Create all Necessary Temporary directories
+    # create temp directory for the repo
+    repo_dir = tmp_path / 'content_repo'
```

Comment:
i think it would make more sense to use here `tmpdir` fixture

Reply:
Unless there is an objection, I'm going to leave it as-is for now, since changing it to use the `tmpdir` fixture would require some reworking of the unit test.